src/app/forms/form-workflow/add-edit-workflow-state/add-edit-workflow-state.component.html

```diff
@@ -0,0 +1,104 @@
+                </div>
+                <label for="departmentStateAccess" class="col-2 col-form-label">Department State Access</label>
+                <div class="col">
+                    <select multiple class="form-select" id="departmentStateAccess" formControlName="departmentStateAccess">
+                        <option *ngFor="let department of Departments" [ngValue]="department.id">
+                            {{department.name}}
+                        </option>
+                    </select>
+                </div>
+            </div>
+            <div class="row">
+                <div class="form-check form-switch col-6">
+                    <label for="sendBackOption" class="col-4 col-form-label">Send Back Option Available</label>
+                    <input class="form-check-input" type="checkbox" id="sendBackOption" formControlName="sendBackAvailable">
+                </div>
+                <label for="sendBackTo" class="col-2 col-form-label">Send Back To</label>
+                <div class="col">
+                    <div class="form-check form-check-inline">
+                        <input class="form-check-input" type="radio" value="initiator" id="inititator" formControlName="sendBackTo">
+                        <label class="form-check-label" for="inititator">Initiator</label>
+                    </div>
+                    <div class="form-check form-check-inline">
+                        <input class="form-check-input" type="radio" value="roleGroup" id="roleGroup" formControlName="sendBackTo">
+                        <label class="form-check-label" for="roleGroup">Role Group</label>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="form-check form-switch col-6">
+                    <label for="triggerEmail" class="col-4 col-form-label">Trigger Notification Via Email</label>
+                    <input class="form-check-input" type="checkbox" id="triggerEmail" formControlName="triggerEmail">
+                </div>
+
+                <div class="form-check form-switch col-6">
+                    <label for="eSignRequired" class="col-4 col-form-label">E-Signature Required</label>
+                    <input class="form-check-input" type="checkbox" id="eSignRequired" formControlName="eSignRequired">
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="form-check form-switch col-6">
+                    <label for="triggerSMS" class="col-4 col-form-label">Trigger Notification Via SMS</label>
+                    <input class="form-check-input" type="checkbox" id="triggerSMS" formControlName="triggerSMS">
+                </div>
+            </div>
+            <div class="row btn_container">
+                <div>
+                    <button class="btn" (click)="Cancel()">Cancel</button>
+                    <button class="btn save_button" type="submit"> Save State</button>
```

Comment:
Lets use same call to action button style everywhere. We should use $main-teal for buttons and $main-teal-dark on hover.